.pylintrc

```diff
@@ -1,556 +1,8 @@
 [MASTER]
 
-# A comma-separated list of package or module names from where C extensions may
-# be loaded. Extensions are loading into the active Python interpreter and may
-# run arbitrary code
-extension-pkg-whitelist=
-
-# Add files or directories to the blacklist. They should be base names, not
-# paths.
-ignore=CVS
-
-# Add files or directories matching the regex patterns to the blacklist. The
-# regex matches against base names, not paths.
-ignore-patterns=
-
-# Python code to execute, usually for sys.path manipulation such as
-# pygtk.require().
-#init-hook=
-
-# Use multiple processes to speed up Pylint.
-jobs=1
-
-# List of plugins (as comma separated values of python modules names) to load,
-# usually to register additional checkers.
-load-plugins=
-
-# Pickle collected data for later comparisons.
-persistent=yes
-
-# Specify a configuration file.
-#rcfile=
-
-# When enabled, pylint would attempt to guess common misconfiguration and emit
-# user-friendly hints instead of false-positive error messages
-suggestion-mode=yes
-
-# Allow loading of arbitrary C extensions. Extensions are imported into the
-# active Python interpreter and may run arbitrary code.
-unsafe-load-any-extension=no
-
+ignore-patterns = ^[A-Z], _imports_  # Dash auto-generated files - ignore them
```

Comment:
Nice solution! Our case of having these special components is probably seldom, but I think we should document somewhere that non-React-based/special components must start with a lowercase letter.

Reply:
Good point. Not sure where we should add the documentation in order to ensure it is found when needed :thinking: Can also hope, if adding more non-React based components, the developer will follow the existing patterns of the non-generated files (`graph.py` and `flexbox.py`) wrt. lower case.